Sources/FunctionalTableData/Accessibility.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Accessibility.swift
+//  FunctionalTableData
+//
+//  Created by Geoffrey Foster on 2020-04-28.
+//  Copyright © 2020 Shopify. All rights reserved.
+//
+
+import UIKit
+
+public struct Accessibility: Equatable {
+	public var identifier: String?
+	public var userInputLabels: [String]?
+	
+	public init(identifier: String? = nil, userInputLabels: [String]? = nil) {
+		self.identifier = identifier
+		self.userInputLabels = userInputLabels
+	}
+	
+	internal func with(identifier: String) -> Accessibility {
```

Comment:
This feels a little misleading to me: UIKit's `with` methods (e.g. `UIFont.withSize`) and our own `with` methods (e.g. in Polaris.FontBook) always override the original value, whereas this specifies a fallback when the field doesn't already have a value.

Maybe just `with(defaultIdentifier:)`?